generator/src/main/java/com/scottlogic/deg/generator/generation/StandardFieldValueSourceEvaluator.java

```diff
@@ -14,23 +14,23 @@
 public class StandardFieldValueSourceEvaluator implements FieldValueSourceEvaluator {
```

Comment:
Just a point of note #587 has changes in this file too, so there may be merge conflicts. Whomever merges first should work with the other person to review the conflicts.